minos/common/model/dto.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from __future__ import (
+    annotations,
+)
+
+import typing as t
+
+from minos.common.model.declarative.abc import (
+    MinosModel,
+    ModelField,
+)
+from minos.common.protocol import (
+    MinosAvroProtocol,
+)
+
+
+class DataTransferObject(MinosModel):
```

Comment:
Also, you must inherit the `DynamicModel` class, that already provides the `from_avro_bytes` method that you need.

Reply:
Done!